app/controllers/events/view/tickets/orders.js

```diff
@@ -1,11 +1,15 @@
 import Controller from '@ember/controller';
 import { run } from '@ember/runloop';
+import { computed } from '@ember/object';
 
 export default Controller.extend({
   isLoadingcsv: false,
 
   isLoadingpdf: false,
 
+  showDeletedRoute: computed('authManager.currentUser.isAdmin', 'authManager.currentUser.isSuperAdmin', function() {
+    return this.get('authManager.currentUser.isSuperAdmin') || this.get('authManager.currentUser.isAdmin');
```

Comment:
@uds5501 Why are you watching `currentUser.isAdmin` ?

Reply:
@kushthedude because only admin should be getting to see the deleted orders (see the issue)

Reply:
@uds5501 By admin , The issue i thing meant only the super-admin , I think .

Reply:
@CosmicCoder96 opinion?

Reply:
@uds5501 @kushthedude IMO we are referring both admin and super admin here. Why not admin?

Reply:
both should see it.